doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -85,7 +85,13 @@ The data is transmitted to nRF Cloud.
 
 Before you start using the Thingy:91, it is recommended that you update the application firmware to :ref:`asset_tracker_v2`.
 You must also update the modem firmware.
-You can do this through :term:`Universal Serial Bus (USB)` (MCUboot) or an external debug probe by using the Programmer application.
+You can update the Thingy:91 application and modem firmware through the :term:`Universal Serial Bus (USB)` cable using MCUboot.
+The MCUboot is a secure bootloader used to update applications in case you do not have an external debugger.
+Alternatively, you can use an external debug probe.
```

Comment:
@gregersrygg, any updates on the recommended order after we discussed it offline?

Reply:
I had a chat with the product owner now. Since the thingy:91 getting started guide focuses on programming pre-compiled firmware he wanted to keep focus on the MCUboot programming in that guide.

However, in the «Developing with thingy:91» guide, he thinks  we should state that it's recommended to use an external debugger (does not have to be in this PR).

Also, the [list of hardware](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#hardware) in the getting started guide could list an external debugger (or 91DK) and the debug cable as recommended if planning to develop on the thingy:91. Since new customers might use that list to plan what hardware they need for developing a prototype (does not have to be in this PR).

Reply:
Thanks for checking!

Reply:
Thanks @gregersrygg . I will create a new separate PR to cover the 2 points you mentioned above.